tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/utilities/ConfigurationReader.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.di.txma.eventprocessor.bdd.utilities;
+
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.ssm.SsmClient;
+import software.amazon.awssdk.services.ssm.model.GetParameterRequest;
+import software.amazon.awssdk.services.ssm.model.GetParameterResponse;
+import software.amazon.awssdk.services.ssm.model.SsmException;
+
+public class ConfigurationReader {
+
+
+    public static String getSqsUrl() {
+        Region region = Region.EU_WEST_2;
+        SsmClient ssmClient = SsmClient.builder()
+                .region(region)
+                .build();
+        String sqsUrl = new String();
+
+        try {
+            GetParameterRequest parameterRequest = GetParameterRequest.builder()
+                    .name("TxMAAccountsQueueArn")
+                    .build();
+
+            GetParameterResponse parameterResponse = ssmClient.getParameter(parameterRequest);
+            System.out.println("paramenter requested" +parameterResponse.parameter().value());
```

Comment:
Same here

Reply:
This is to get the ssm parameter as we need to keep the url out of public repo